src/controllers/routes/http.routes.ts

```diff
@@ -13,7 +14,7 @@ export async function setupHttpRoutes(context: GlobalContext): Promise<Router<Gl
 
   const router = new Router<GlobalContext>()
 
-  const signedFetchMiddleware = (optional: boolean = false) =>
+  const signedFetchMiddleware = ({ optional = false }: { optional?: boolean } = {}) =>
```

Comment:
I like this type more. We can even type it using the Options from the `@dcl/platform-crypto-middleware`